src/index.js

```diff
@@ -1150,7 +1150,16 @@ export default class Carousel extends React.Component {
                 deltaX={tx}
                 deltaY={ty}
               >
-                {validChildren}
+                {React.Children.map(validChildren, (child, index) => {
+                  const ariaProps =
```

Comment:
@sarmeyer So I am using the same logic as before here but instead of using css `{display: none}` I am using `{aria-hidden: true}`. This way the images are still seen on the DOM keeping  the look it had before but won't be read my the reader. Also, the tabbing order got messed up because I added `tabindex: 1` to all the images. Now its just adding `tabindex` the one the is visible.

Reply:
@ElreyB So I just updated to this version and narrowed down a bug to this change that is highlighted here.

On mobile iPhone devices VoiceOver correctly focuses and reads the FIRST slide, but then skips all the rest of the visible slides. Instead it goes to the next div (i.e. some other button on the page) effectively causing a11y users to miss out on all the other slides.

Any ideas on what needs to be fixed here?